notebooks/tutorials/overland_flow/overland_flow_erosion/ol_flow_erosion_components.ipynb

```diff
@@ -0,0 +1,489 @@
+    "n_sp = 1.0 # slope exponent\n",
+    "I_c = 0.0001 # erosion threshold, m/s\n",
+    "\n",
+    "# Run-control parameters\n",
+    "rain_duration = 240.0 # duration of rainfall, s\n",
+    "run_time = 480.0  # duration of run, s\n",
+    "dt = 10.0 # time-step size, s\n",
+    "dem_filename = '../hugo_site_filled.asc'\n",
+    "\n",
+    "# Derived parameters\n",
+    "num_steps = int(run_time / dt)\n",
+    "\n",
+    "# set up arrays to hold discharge and time\n",
+    "time_since_storm_start = np.arange(0.0, dt * (2 * num_steps + 1), dt)\n",
+    "discharge = np.zeros(2 * num_steps + 1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Read the DEM file as a grid with a 'topographic__elevation' field\n",
+    "(grid, elev) = read_esri_ascii(dem_filename, name='topographic__elevation')\n",
+    "\n",
+    "# Configure the boundaries: valid right-edge nodes will be open; \n",
+    "# all NODATA (= -9999) nodes will be closed.\n",
+    "grid.status_at_node[grid.nodes_at_right_edge] = grid.BC_NODE_IS_FIXED_VALUE\n",
+    "grid.status_at_node[np.isclose(elev, -9999.)] = grid.BC_NODE_IS_CLOSED"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slp_mag, slp_x, slp_y = slope_magnitude_at_node(grid, elev)\n",
+    "grid.add_field('topographic__slope', slp_mag, at='node', clobber=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Instantiate the component\n",
+    "olflow = KinwaveImplicitOverlandFlow(grid,\n",
+    "                                     runoff_rate=R, \n",
+    "                                     roughness=n,\n",
+    "                                     depth_exp=dep_exp\n",
+    "                                    )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "dle = DetachmentLtdErosion(grid,\n",
+    "                           K_sp=K_sp,\n",
+    "                           m_sp=m_sp,\n",
+    "                           n_sp=n_sp,\n",
+    "                           entrainment_threshold=I_c\n",
+    "                          )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "starting_elev = elev.copy()\n",
+    "\n",
+    "for i in range(num_steps):\n",
+    "    olflow.run_one_step(dt)\n",
+    "    dle.run_one_step(dt)\n",
+    "    slp_mag[:], slp_x, slp_y = slope_magnitude_at_node(grid, elev)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "imshow_grid(grid, starting_elev - elev, colorbar_label='cumulative erosion (m)')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
I'm not seeing the link to "more Landlab tutorials" at the bottom of this notebook.

Reply:
Weird, it's there...